src/init.cpp

```diff
@@ -513,6 +513,7 @@ std::string HelpMessage(HelpMessageMode mode)
     if (showDebug) {
         strUsage += HelpMessageOpt("-fedpegscript=<hex>", _("Change federated peg to use a different script.") +
             " " + _("This creates a new chain with a different genesis block."));
+        strUsage += HelpMessageOpt("-parentconfirmationdepth", strprintf(_("Pegin claims must be this deep to be considered valid. (default: %d)"), 8));
```

Comment:
Maybe make this a `DEFAULT_PARENT_CONFIRMATION_DEPTH` const in `script.h`?

Name is a tiny bit ambiguous, but maybe ok. It's not immediately obvious it refers to peg-ins (I was confused until I read through the tests). Maybe `-peginconfirmationdepth`?